src/custodian/src/custodian_db_checker.erl

```diff
@@ -132,7 +132,7 @@ get_dbs() ->
 
 
 get_users_db() ->
-    UsersDb = config:get("couch_httpd_auth", "authentication_db", "users"),
+    UsersDb = config:get("couch_httpd_auth", "authentication_db", "_users"),
```

Comment:
Any reason to not use: https://github.com/apache/couchdb/blob/3.x/src/chttpd/src/chttpd_auth_cache.erl#L188-L189

Reply:
That particular function is not exported. If you prefer, I can also export `dbname/1` from chttpd_auth_cache and then use it.

Reply:
Fixed up with exported `dbname/0`